src/backend/cpu/kernel/fftconvolve.hpp

```diff
@@ -114,11 +114,8 @@ void complexMultiply(Array<T> packed, const af::dim4 sig_dims, const af::dim4 si
                         T c = in2_ptr[ridx];
                         T d = in2_ptr[iidx];
 
-                        T ac = a*c;
-                        T bd = b*d;
-
-                        out_ptr[ridx] = ac - bd;
-                        out_ptr[iidx] = (a+b) * (c+d) - ac - bd;
+                        out_ptr[ridx] = a*c - b*d;
+                        out_ptr[iidx] = a*d + b*c; // (a+b) * (c+d) - ac - bd;
```

Comment:
Remove the comment. The actual computation is a standard way to do it. The comment is more confusing.

Also remove it from everywhere else in this PR.